composeApp/src/androidInstrumentedTest/kotlin/org/ooni/probe/uitesting/screenshots/AutomateScreenshotsTest.kt

```diff
@@ -234,4 +245,233 @@ class AutomateScreenshotsTest {
                 Screengrab.screenshot("16-about")
             }
         }
+
+    @Test
+    fun results() =
+        runTest {
+            skipOnboarding()
+            setupTestResults()
+            start()
+            with(compose) {
+                wait { onNodeWithContentDescription(Res.string.app_name).isDisplayed() }
+
+                clickOnText(Res.string.TestResults_Overview_Title)
+
+                wait { onNodeWithText("Websites").isDisplayed() }
+
+                Screengrab.screenshot("17-results")
+
+                clickOnText("Websites")
+
+                wait(10.seconds) { onNodeWithText("https://z-lib.org/").isDisplayed() }
+
+                Thread.sleep(1000)
+                Screengrab.screenshot("18-websites-results")
+                Thread.sleep(1000)
+
+                clickOnText("https://z-lib.org/")
+
+                checkTextAnywhereInsideWebView("https://z-lib.org/")
+
+                Screengrab.screenshot("19-website-measurement-anomaly")
+
+                clickOnContentDescription(Res.string.Common_Back)
+                wait { onNodeWithText("Websites").isDisplayed() }
+                clickOnContentDescription(Res.string.Common_Back)
+                wait { onNodeWithText("Websites").isDisplayed() }
+                clickOnText("Performance")
+                wait { onNodeWithText("DASH Streaming Test").isDisplayed() }
+                clickOnText("DASH Streaming Test")
+
+                checkTextAnywhereInsideWebView("2160p (4k)")
```

Comment:
We may want to change this and strings file to pull the actual text, since this would not work in any other language.